jquery.flot.touchNavigate.js

```diff
@@ -48,9 +48,6 @@ can set the default in the options. */
     'use strict';
 
     var options = {
-        pan: {
```

Comment:
Because it was overriding the value of enableTouch that came from the parent. And it shouldn't.